src/support.js

```diff
@@ -206,20 +206,22 @@ const supportCommands = () => {
   Cypress.Commands.add('checkAllResults', checkAllResults)
 
   /**
-   * Create a menu item
+   * Custom Cypress command to create a menu item in Joomla.
    *
-   * @memberof cy
+   * @memberof Cypress.Commands
    * @method createMenuItem
-   * @param {string} menuTitle
-   * @param {string} menuCategory
-   * @param {string} menuItem
-   * @param {string} menu
-   * @param {string} language
-   * @param {string} extension
-   * @returns Chainable
+   * @param {string} menuTitle – The title of the menu item to be created.
+   * @param {string} menuCategory – The category of the menu item.
+   * @param {string} menuItem – The used menu item type (e.g. 'Articles').
+   * @param {string} [menu='Main Menu'] – The used menu item destination (e.g. 'Featured Articles').
+   * @param {string} [language='All'] - Menu item language as name (e.g. 'Czech (Čeština)') or tag (e.g. 'cs-CZ').
+   *
+   * The 'language' parameter is only used for multilingual websites where the language selection is visible.
+   * 
+   * @returns {Cypress.Chainable}
```

Comment:
No return here

Reply:
Yes, this is correct. I have the method documentation rewritten:
```
@returns {void} - It does not return any value, but it is chainable with other Cypress commands.
```